### DISCUSSION TITLE
Custom "Edit this page" repository links

### DISCUSSION THREAD

**Discussion by sccmcca** [squidfunk/mkdocs-material#3242] [Ideas]
Is there a configuration or plugin that would allow to define custom links to different source repositories, per page?

In my case I have a bash script that fetches markdown files from multiple repositories and mkdocs renders them as pages.

If a user wants to edit, I'd like them to be able to click the "Edit this page" button and be directed to the original file on the original repository (different from my site repository). I have no problem defining these manually as I have a minimal number of pages.

Any hints on this? Thanks!

**Comment by squidfunk** [comment]
You can change the [`edit_uri`](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#edit-button) in `mkdocs.yml`, but that would link to the repository you defined in `repo_url`. There's currently no other option. This is a limitation of MkDocs, not Material for MkDocs.
